src/main/java/com/cairn/waypoint/dashboard/endpoints/homework/UpdateHomeworkResponsesEndpoint.java

```diff
@@ -47,144 +43,110 @@
 @Tag(name = "Homework")
 public class UpdateHomeworkResponsesEndpoint {
 
-  public static final String PATH = "/api/homework/{homeworkId}";
-  private final HomeworkDataService homeworkDataService;
+  public static final String PATH = "/api/protocol/{protocolId}";
   private final S3FileUpload s3FileUpload;
-  private final ProtocolStepDataService protocolStepDataService;
   private final ProtocolDataService protocolDataService;
+  private final HomeworkResponseDataService responseDataService;
 
   @Value("${waypoint.dashboard.s3.homework-response-key-prefix}")
   private String homeworkResponseKeyPrefix;
 
-  public UpdateHomeworkResponsesEndpoint(HomeworkDataService homeworkDataService,
-      S3FileUpload s3FileUpload, ProtocolStepDataService protocolStepDataService,
-      ProtocolDataService protocolDataService) {
-    this.homeworkDataService = homeworkDataService;
+  public UpdateHomeworkResponsesEndpoint(S3FileUpload s3FileUpload, ProtocolDataService protocolDataService,HomeworkResponseDataService responseDataService ) {
```

Comment:
Make sure your add spaces between arguments for readability. I should send you the style guide I use so that it does this for you on save